Source/storm/storm.cpp

```diff
@@ -215,6 +215,41 @@ void setIniFloat(const char *keyname, const char *valuename, float value)
 	getIni().SetDoubleValue(keyname, valuename, value);
 }
 
+/**
+ * @brief Check item exists in comma-sparated list
+ */
+bool IniFound(const char *list, const char *item)
+{
+	const char *look = strcasestr(list, item);
+	if (!look)
+		return false;
+
+	look += strlen(item);
+	while (*look == ' ')
+		look++;
+	return (bool) strchr(",;:", *look); // delimters including '\0'
+}
+
+/**
+ * @brief Find needle pattern in haystack, ignoring case
+ */
+const char *strcasestr(const char *haystack, const char *needle)
+{
+	const char *look, *pp, *qq;
+	assert(haystack && needle);
+
+	for ( look = haystack; *look; look++ ) {

```

Comment:
Format your code. There should be no space between the `(` and the members inside the `for`.